### ISSUE TITLE
Typically, $.ajax() is preferred over Razor's Ajax.BeginForm()

### ISSUE THREAD

**Issue by WtheFourth** [newtonrd/InventoryManagement-BT#6]
See: https://github.com/newtonrd/InventoryManagement-BT/blob/master/InventoryManagement-BT/Views/Inventory/TakeInventory.cshtml#L9

This could be refactored to be a standard HTML form (potentially, you'd want to give it its own class, such as `.take-inventory-form`), with the following jQuery function:

```javascript
$('.take-inventory-form').on('submit', function(e) {
    e.preventDefault();
    $.ajax({
        url: $(this).attr('action'),
        type: 'POST',
        data: $(this).serialize(),
        success: function(data) {
            HideTakeInventoryDescription(); // might not even need to be a function, seems simple.
            $('#search-results').html(data);
        },
        error: function(xhr) {
            HandleError(xhr);
        };
    });
});
```

**Comment by WtheFourth** [comment]
A similar thing could be implemented [here](https://github.com/newtonrd/InventoryManagement-BT/blob/master/InventoryManagement-BT/Views/Inventory/ViewInventory.cshtml#L10).

**Comment by WtheFourth** [comment]
And [here](https://github.com/newtonrd/InventoryManagement-BT/blob/master/InventoryManagement-BT/Views/Inventory/_addInventory.cshtml#L13) and [here](https://github.com/newtonrd/InventoryManagement-BT/blob/master/InventoryManagement-BT/Views/Inventory/_modifyAsset.cshtml#L21) and, slightly differently (using a GET for the type rather than POST), [here](https://github.com/newtonrd/InventoryManagement-BT/blob/master/InventoryManagement-BT/Views/Inventory/_searchResults.cshtml#L66) and [here](https://github.com/newtonrd/InventoryManagement-BT/blob/master/InventoryManagement-BT/Views/Inventory/_viewInventorySearchResults.cshtml#L27).

**Comment by newtonrd** [comment]
Closed by #21
